tools/devcluster.yaml

```diff
@@ -2,9 +2,9 @@
 startup_input: "p"
 
 commands:
-  p: make -C harness clean build  # rebuild Python
-  w: make -C webui build          # rebuild Webui
-  c: make -C docs build           # rebuild doCs
+  p: make -C bindings build && make -C harness clean build  # rebuild Python
+  w: make -C bindings build && make -C webui build          # rebuild Webui
```

Comment:
make -C bindings build/python && make -C harness clean build
make -C bindings build/typescript && make -C webui build

Reply:
I don't feel strongly about this